compose/container.py

```diff
@@ -212,6 +212,21 @@ def restart(self, **options):
     def remove(self, **options):
         return self.client.remove_container(self.id, **options)
 
+    def create_exec(self, command, **options):
+        privileged = options.get("privileged")
+        user = options.get("user")
+        tty = options.get("tty")
+        return self.client.exec_create(
+            self.id, command, privileged=privileged, tty=tty, stdin=tty, user=user)
+
+    def start_exec(self, exec_id, **options):
+        detach = options.get("detach")
+        tty = options.get("tty")
+        return self.client.exec_start(exec_id, detach=detach, tty=tty)
```

Comment:
Small thing, but:  in these two methods, rather than extracting the kwargs we recognise and silently discarding anything else, why don't we just pass the whole dict off to `exec_create`/`exec_start`? That way if, a kwarg gets mistyped in future code changes, we find out about it immediately (because the docker-py method will raise an error).